tools/tidb-binlog-kafka.md

```diff
@@ -109,8 +109,8 @@ cd tidb-binlog-latest-linux-amd64
 
 ### Deploy Pump using TiDB-Ansible
 
-- If you have not deployed the Kafka cluster, use the [Kafka-Ansible](https://github.com/pingcap/thirdparty-ops/tree/master/kafka-ansible) to deploy.
-- When you deploy the TiDB cluster using [TiDB-Ansible](https://github.com/pingcap/tidb-ansible), edit the `tidb-ansible/inventory.ini` file, set `enable_binlog = True`, and configure the `zookeeper_addrs` variable as the ZooKeeper address of the Kafka cluster. In this way, Pump is deployed while you deploy the TiDB cluster.
+- If you have not deployed the Kafka cluster, use the [kafka-ansible](https://github.com/pingcap/thirdparty-ops/tree/master/kafka-ansible) to deploy.
+- When you deploy the TiDB cluster using [tidb-ansible](https://github.com/pingcap/tidb-ansible), edit the `tidb-ansible/inventory.ini` file, set `enable_binlog = True`, and configure the `zookeeper_addrs` variable as the ZooKeeper address of the Kafka cluster. In this way, Pump is deployed while you deploy the TiDB cluster.
```

Comment:
Thanks! But when "Kafka-Ansible" and "TiDB-Ansible" appear as tool names, we usually capitalize the first letter.